acpi.adoc

```diff
@@ -7,18 +7,9 @@ This section defines mandatory and optional ACPI requirements on top of the ACPI
 
 === ACPI High-Level Requirements
 
-A compliant system must:
-
-* Conform to UEFI specification requirements for ACPI:
```

Comment:
You've removed a lot, including aspects that are not covered as *requirements* by any other spec. For example, nothing today requires an RHCT, or specific MADT entries (as there isn't a ACPI section for RISC-V).

This change makes the spec useless because it forces the reader to go on a discovery quest to see what the implicit requirements are.

Reply:
> This doesn't fit the spirit of the spec, which is meant to be concise and useful. The BRS spec /points out/ those aspects which are not necessary clear from the (very large) dependent specs, to avoid ambiguity and the need to reference existing implementations just to chart out a course of action.

I agree that the spirit of the specification is to be concise and useful and this change does just that.

If it is valuable to point out anything in another specification, that does not belong in the normative BRS Specification.  Repeating and slightly rewording things from elsewhere will create ambiguity and give the reader even more text to wade through in order to understand the specification.

Reply:
"Today" is 6.5. 6.6 isn't out and doesn't have a chance at containing everything we'd like ACPI to be precise on.

The BRS specification is not the Dummy's Guide to ACPI and UEFI, but you would be surprised at the quality of implementations when you try to be least helpful to the very people you expect to rely on for your success. I'm fine with moving things into non-normative, but the references need to exist. You can't expect people to read 1200 pages of the ACPI spec and almost 3000 pages of the UEFI spec (and there are other specs too...), and nail down an implementation without so much as a hint in the right direction.

Reply:
The BRS Specification requires ACPI version ">= 6.6".  If that version does not exist, or is not expected to exist, the requirement in the BRS Specification needs to change.  It's hard to have meaningful discussion on this if we can't be clear and consistent about which versions of the dependent specifications we are talking about.

I take no position on whether the extraneous information and restatement of requirement in other specifications is useful or whether or not a guidance document is needed. My point is that the BRS Specification is not the place for it.

If the concern you speak of is the excessive length of the ACPI, UEFI, and other specifications, I question whether making specifications longer is the right solution.

Reply:
Whether you keep 6.5 or 6.6 depends on where we expect this spec to land relative to 6.6 getting ratified. I agree that talking about 6.5 is probably not net useful.

If something is not on track to be in 6.6 - it needs to be here. This may include generic guidance info around the mandatory ACPI tables, for example. When stuff lands in the ACPI it can be removed here. Same goes for the UEFI spec. We're talking about the same set of people working on all of these, so it's not unreasonable.